pr-deploy/README.md

```diff
@@ -0,0 +1,55 @@
+AMP PR Deploy Bot
+===================
+
+A GitHub App deploys a PR branch to a static website when you ask it to. 
+
+Websites are served by a [Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/amp-test-website-1). <br>
+The app is hosted by [Google App Engine](https://amp-pr-deploy-bot.appspot.com) and is installed exclusively on the ampproject/amphtml repository.
+
+How the app works
+----------------
+1. A commit is pushed to a new or existing pull request on ampproject/amphtml.
```

Comment:
Supernit: I don't like periods at the end of lists :P

Reply:
Counter-argument: When list items contain multiple sentences, I think they're reasonable

Reply:
Yeah I'm gonna vote keeping the periods.

Reply:
😢